### PR TITLE
add Codeanywhere

### PR DESCRIPTION
**Pull request by zhang-yuan**

### CODE REVIEW

**Comment by styfle** [comment]
Thanks for the PR!

Can you change the description to start with "Run" like the other entries so that we know what kind of code you can run/execute with this tool and what sets it apart?

**Comment by zhang-yuan** [comment]
Modified.
Sorry for the bothering:)

File: README.md [hunk 1]
```diff
@@ -19,8 +19,7 @@ A curated list of awesome online implementations of integrated development envir
 * [CodeSandbox](https://codesandbox.io) <img title="Open Source" width="16" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/github.svg" /> - Run javascript with starting templates for React, Preact, and Vue with some intellisens + lint error checking
 * [WebpackBin](https://www.webpackbin.com) <img title="Open Source" width="16" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/github.svg" /> - Run webpack bundling with starting templates for Babel ES2015 or TypeScript
 * [Snack Expo](https://snack.expo.io) - Run your React Native javascript in the browser and on your mobile iOS or Android
-* [Codeanywhere](https://codeanywhere.com/) - A complete toolset for web development. Enabling you to edit, collaborate and run your projects on any device.
-
+* [Codeanywhere](https://codeanywhere.com/) - Run a large number of languages on a container comes in numerous pre-built envirnment with a terminal
```
Comment (styfle) [inline README.md]: Typo here. I think `envirnment ` should be `environment`.

**Comment by styfle** [comment]
Thanks 🎉

**Pull request merged**